modules/assim.sequential/DESCRIPTION

```diff
@@ -1,7 +1,8 @@
 Package: PEcAnAssimSequential
 Type: Package
 Title: PEcAn Functions Used for Ecological Forecasts and Reanalysis
-Version: 1.8.0.9000
+Version: 1.7.2
+Date: 2021-10-04
```

Comment:
Version and Data are wrong, please revert

Reply:
addressed in [b1bd57f](https://github.com/PecanProject/pecan/pull/3321/commits/b1bd57fa514416e771f7381d6334cc0a25dc58ec) commit